.github/workflows/ci.yml

```diff
@@ -18,14 +18,18 @@ jobs:
         with:
           fetch-depth: 2
 
-      - uses: oven-sh/setup-bun@v1
-        with:
-          bun-version: latest
+      - name: Install pnpm
+        run: |
+          corepack enable
+          corepack prepare pnpm@latest --activate
 
       - name: Install dependencies
-        run: bun install
+        run: pnpm install
+
+      - name: Lint files
+        run: pnpm lint
 
-      - name: Build
+      - name: Build project
         env:
```

Comment:
Let's remove the `env` in this CI, seems like this was accidentally pasted from another project. This shouldn't affect anything as this project doesn't use `turso`